distributions/demo-google-deployment/bin/build_docker_image.sh

```diff
@@ -59,11 +60,13 @@ fi
 BINARY=$1
 ENV=$2
 PROJECT_ID_SUFFIX=$3
-SRC_DIR="portability-$BINARY"
+SRC_DIR=$BINARY
```

Comment:
Are we using SRC_DIR anywhere anymore? can we just use BINARY?

Reply:
Good catch, done